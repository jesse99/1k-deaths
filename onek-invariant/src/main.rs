mod invariant;
mod state;
mod tests;

fn main() {
    println!("TODO: validate State");
}
