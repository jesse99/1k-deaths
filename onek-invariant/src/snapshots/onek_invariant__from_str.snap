---
source: onek-invariant/src/main.rs
expression: state.to_snapshot(&test)
---
###
#@#
###

